perl-lib/OESS/lib/OESS/DB/VRF.pm

```diff
@@ -145,20 +145,64 @@ sub delete_endpoints {
     return $ok;
 } 
 
+sub find_available_unit{
+    my %params = @_;
+    my $db = $params{'db'};
+    my $interface_id = $params{'interface_id'};
+    my $tag = $params{'tag'};
```

Comment:
tag and inner_tag aren't used in this method. they can be removed

Reply:
I left these for future use (if we need them later)